Bai1/PhanSo.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bai1
+{
+    internal class PhanSo
+    {
+        public float tuSo1 { get; set; }
```

Comment:
biến theo pascal case nhé
**When naming public members of types, such as fields, properties, events, methods, and local functions, use pascal casing.**